sos/plugins/memory.py

```diff
@@ -36,4 +36,8 @@ def setup(self):
             "lsmem -a -o RANGE,SIZE,STATE,REMOVABLE,ZONES,NODE,BLOCK"
         ])
 
+        # slabtop -o will hang if not handed a tty via stdin and our timeout
+        # handling removes the tty for any command run with it
+        self.add_cmd_output("slabtop -o", timeout=None)
```

Comment:
Since #1909 is merged, is it worth changing `timeout=None` to `foreground=True`?

Reply:
Yep, was either going to update this one once #1909 got merged or vice versa. Will update this PR later today.